apps/cyberstorm-nextjs/next.config.js

```diff
@@ -2,6 +2,10 @@ const withPreconstruct = require("@preconstruct/next");
 
 /** @type {import('next').NextConfig} */
 const nextConfig = withPreconstruct({
+  basePath: "/cyberstorm",
+  images: {
+    path: "/cyberstorm/_next/image",
```

Comment:
What does this actually do? I couldn't find `images.path` in [the documentation](https://nextjs.org/docs/app/api-reference/next-config-js/images), and the MM screenshot seems to load fine without it. Config files are probably the last place where we want to have leftover obsolete code, because no one dares to touch them unnecessarily. If this indeed is an undocumented feature that serves some purpose, maybe add a comment about it.

Reply:
Huh, yeah, I'm not sure what docs I've been looking at since I couldn't find it either now. Since it works without it just fine, I'll remove it 🤔